tutorials/multi/1_graph_classification.py

```diff
@@ -0,0 +1,241 @@
+"""
+Single Machine Multi-GPU Minibatch Graph Classification
+=======================================================
+
+In this tutorial, you will learn how to use multiple GPUs in training a
+graph neural network (GNN) for graph classification. This tutorial assumes
+expertise in GNNs for graph classification and we recommend you to check
+:doc:`Training a GNN for Graph Classification <../blitz/5_graph_classification>` otherwise.
+
+(Time estimate: 8 minutes)
+
+To use a single GPU in training a GNN, we need to put the model, graph(s), and other
+tensors (e.g. labels) on the same GPU:
+"""
+
+"""
+import torch
+
+# Use the first GPU
+device = torch.device("cuda:0")
+model = model.to(device)
+graph = graph.to(device)
+labels = labels.to(device)
+"""
+
+###############################################################################
+# The node and edge features in the graphs, if any, will also be on the GPU.
+# After that, the forward computation, backward computation and parameter
+# update will take place on the GPU. For graph classification, this repeats
+# for each minibatch gradient descent.
+#
+# Using multiple GPUs allows performing more computation per unit of time. It
+# is like having a team work together, where each GPU is a team member. We need
+# to distribute the computation workload across GPUs and let them synchronize
+# the efforts regularly. PyTorch provides convenient APIs for this task with
+# multiple processes, one per GPU, and we can use them in conjunction with DGL.
```

Comment:
`and we can use them in conjunction with DGL.` -> We follow those standard practices in DGL also.?

Reply:
If we want to say "practices", then these are broader than particular APIs.

Reply:
I just feel you can start another sentence for the `we can use them in conjunction with DGL`. And `in conjunction with DGL` reads tedious to me

Reply:
How about ", and we can use them in conjunction with DGL." -> ". They are compatible with DGL."?